### ISSUE TITLE
Bug in Integrity Verdicts Display

### ISSUE THREAD

**Issue by MeowDump** [1nikolas/play-integrity-checker-app#55]
I noticed a bug in your Play Integrity API Checker app. The app shows "MEETS DEVICE INTEGRITY", but when I check through the Play Store developer options on the same device, it shows "MEETS STRONG INTEGRITY" instead.

It looks like the app might not be handling or displaying the full list of integrity levels properly. Here’s a snippet from the JSON response:

```
"deviceIntegrity": {
    "deviceRecognitionVerdict": [
        "MEETS_DEVICE_INTEGRITY"
    ]
}
```

But since the Play Store is reporting MEETS_STRONG_INTEGRITY, I believe this should also be reflected in the app's result.

Let me know if you need any other details

[PIAC.json](https://github.com/user-attachments/files/21859663/PIAC.json)

https://drive.google.com/file/d/1AAL36kitolIl43-7JGHaH0Ysw2qF_9xB/view?usp=drivesdk

**Comment by gavdoc38** [comment]
It needs to show this in app info or by using pm command to check installer source. Simplest thing would be to uninstall and install from Play Store.

<img width="1440" height="507" alt="Image" src="https://github.com/user-attachments/assets/5768ecfa-d9b5-4ef5-bb69-df0a6e4176c0" />

**Comment by 1nikolas** [comment]
If that's the case I'll just implement #56

**Comment by MeowDump** [comment]
> Did you install the app from play store? Sideloading (even when restoring a backup) gets picked up by the integrity api and returns different results..

Thanks for the quick response! 
You were right. I had originally restored the app from a backup. After uninstalling and reinstalling it directly from the Play Store, the integrity check now shows the correct Play Integrity verdicts in the app.

Problem solved 👍

**Comment by abdoelsisi398-del** [comment]
https://github.com/1nikolas/play-integrity-checker-app/issues/55
